app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -91,6 +100,16 @@ class BrowserFragment : WebFragment(), CursorEvent {
         })
     }
 
+    override fun onEvent(event: BrowserNavigationOverlay.NavigationEvent, value: String?, autocompleteResult: InlineAutocompleteEditText.AutocompleteResult?) {
```

Comment:
I also think `onEvent` should be more descriptive (if you keep it) – if I was reading the code, all I would know about this fn is it's `BrowserFragment.onEvent` – what does that do? What kind of events? What happens when there are `onEvent` name collisions?